OneCommunityGlobal/HighestGoodNetworkApp
# Masasa_Visualizations for Teams

# Description
<img width="734" alt="Screenshot 2023-12-10 at 10 12 45 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73863365/8ddfe07d-2a99-46b6-99ad-bad0056cdd83">

This PR aims at developing the requested features as described in the Picture above. Please see the video below for how to test the implementation.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73863365/5561407e-71c7-4860-ab10-35be54d76248

## Related PRS (if any):
None
…

## Main changes explained:
- Intergrated  the chart using recharts library into weeklySummariesReport.jsx…
- created a new component called teamChart.jsx and its styling teamChart.css to display the table of the members…
- integrated the logic to enable the table and chart to dynamically reflect properties of the selected teams …
- added the ability to visit a member's profile by clicking on thier name in the table
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin or owner
5. go to dashboard→ Reports→ weekly Reports→…
6. verify that the implementation works as shown in the video
